ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SignatureIntegrationForm.tsx

```diff
@@ -63,6 +73,31 @@ const VerificationExpandableSection = ({ toggleText, children }): ReactElement =
     );
 };
 
+function regularExpressionIcon(): ReactElement {
+    return (
+        <Popover
+            bodyContent={
+                <div>
+                    <a href="https://golang.org/s/re2syntax" target="_blank" rel="noreferrer">
+                        See RE2 syntax here
```

Comment:
Where is here :)

Reply:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/MachineAccessIntegrationForm.tsx#L231-L238

Reply:
I've named it `RE2 Syntax reference` now, hope this makes things somewhat clearer?